crates/ln-dlc-node/src/node/dlc_manager.rs

```diff
@@ -39,6 +40,15 @@ pub fn build(
     let oracle_pubkey = p2pdoracle.get_public_key();
     let oracles = HashMap::from([(oracle_pubkey, p2pdoracle)]);
 
+    // FIXME: We need to do this to ensure that we can upgrade `Node`s from LDK 0.0.114 to 0.0.116.
+    // We should remove this workaround as soon as possible.
+    if let Err(e) = storage.get_chain_monitor() {
+        tracing::error!("Failed to load DLC ChainMonitor from storage: {e:#}");
```

Comment:
🤔 shouldn't this rather be a warning, since we are able to recover from it?

Reply:
I want to be scared every time we see this message, because we really should remove the workaround ASAP.